markdown.dtx

```diff
@@ -23176,10 +23187,18 @@ end
 %
 % \end{markdown}
 %  \begin{macrocode}
+\ExplSyntaxOn
 \def\markdownLATEXStrongEmphasis#1{%
+  \str_if_in:NnTF {
+    \f@series
+    { b }
+    { \textnormal{#1} }
+    { \textbf{#1} }
+  }
```

Comment:
The `{` after `\str_if_in:NnTF` and the `}` below `{ \textbf{#1} }` should go away.